src/core/getters/scalar.mock.ts

```diff
@@ -125,13 +125,7 @@ export const getMockScalar = async ({
         );
         const enumValue = enumImp?.name || name;
         return {
-          value: `[...Array(faker.datatype.number({min:1, max: ${enums.length}}))].reduce(({values, enums}) => {
-            const newValue = enums[faker.datatype.number({min:0, max: enums.length - 1})];
-            return {
-              values: [...values, newValue],
-              enums: enums.filter((v: ${enumValue}) => newValue !== v)
-            }
-          },{ values: [], enums: Object.values(${enumValue})}).values`,
+          value: `faker.random.arrayElements(Object.values(${enumValue}), faker.datatype.number({min:1, max: ${enums.length}}))`,
```

Comment:
I am not sure we need the second argument with the new API. I think we can remove it no?

Reply:
Yes, actually. I had it in my mind the second arg was required. I just force-pushed now :)